README.md

```diff
@@ -9,6 +9,14 @@ A Config Controller
 Using YAML files *endpoints* can be defined to be exposed or called on specified conditions. The call of an endpoint results in *coco* calling the same endpoint on a given set of hosts. *coco* can be configured to check or compare the results and based on that export [prometheus](prometheus.io) alerts, write [slack](slack.com) messages or call other endpoints.  
 The values passed to *endpoints* can be marked as part of a global state, which is tracked and kept in sync on the hosts.
 
+## Metrics
+Prometheus metrics are exported at a port specified in the config (`metrics_port`).
+These are:
+- `coco_requests` (labels: `['endpoint']`): Total number of requests received per endpoint.
+  **NB** we intend to change this one to a metric that tracks dropped requests.
+- `coco_results` (labels: `['endpoint', 'host', 'port', 'status']`):
```

Comment:
This metric doesn't really count "results". How about "forward" or "calls"?

Reply:
I can change it to "calls"